### ISSUE TITLE
Support for key modifier prefixes for shortcut keys

### ISSUE THREAD

**Issue by mrsean2k** [xgi/castero#144]
Is there any way to allow modifier keys such as CTRL to be used in the interface?

Aware this might just be my ignorance if this is supported in the configuration file.

**Comment by xgi** [comment]
Yeah, this is not currently supported with the config settings. I'll have to look into more how this works with the UI library, but I'll see if I can add this. I don't think I could explicitly handle key-sequences but I should be able to differentiate inputs like `^M` for `ctrl-shift-m`. Will need to experiment more. Thanks!

**Comment by tistatos** [comment]
It seems that it is possible to use `Shift` as a modifier key.
This is simply made by using the uppercase-letter in the config file.

For instance:
```
# Show the help menu.
# default: h
key_help = H
```
**NOTE:** Having caps-lock active will have the same effect
